style.css

```diff
@@ -0,0 +1,44 @@
+@import url('https://fonts.googleapis.com/css2?family=Rubik&display=swap');
+html {
+
+}
+body {text-align: center;
+    font-family: 'Rubik', sans-serif;
```

Comment:
This line is not properly indented. I encourage you to make a habit of using two spaces for your indentation.